starlark/testdata/math.star

```diff
@@ -0,0 +1,43 @@
+# Tests of math module.
+
+load('math.star', 'math')
+load('assert.star', 'assert')
+
+def assert_near(actual, desired, threshold):
```

Comment:
This function doesn't actually assert, and can be simplified to:
```
def near(got, want, threshold):
  return math.abs(got-want) < threshold  
```

That said, we'd get more informative error messages if they were of the form "want x±k, got y".